### PR TITLE
Feature 100 additional rl

### PR DESCRIPTION
**Pull request by VikasChidananda**
-  Add `TD3` as an additional RL agent
-  Add example notebook for using `TD3`
-  Add Wandb example

### CODE REVIEW

File: src/multi_controller.jl
Comment (Webbah) [inline src/multi_controller.jl:1]: Are these changes (episode & tstep) still needed or is that from debugging?

Can the comments @show be deleted?

**Comment by Webbah** [comment]
@janstenner, are the changes in the Poject.toml ok to your mind?

File: src/multi_controller.jl
Comment (VikasChidananda) [reply src/multi_controller.jl:1]: oh yes, definitely. It was just for debugging purposes.

Comment (VikasChidananda) [reply src/multi_controller.jl:1]: But `tstep` and `episode` are for the recording and showing in wandb in realtime. They are required.

Comment (Webbah) [reply src/multi_controller.jl:1]: ok, did we decide to put this functionality into our code?

**Review by Webbah — APPROVED** [review]

**Comment by janstenner** [comment]
> @janstenner, are the changes in the Poject.toml ok to your mind?

yes, looking good to me

**Pull request merged**